packages/lesswrong/lib/voting/votingSystemTypes.ts

```diff
@@ -14,8 +15,9 @@ export type VotingPropsDocument = CommentsList | PostsWithVotes | RevisionMetada
   post?: PostsWithNavigation | PostsWithNavigationAndRevision;
 };
 
-export interface NamesAttachedReactionsCommentBottomProps extends CommentVotingComponentProps<CommentsList> {
+export interface NamesAttachedReactionsCommentBottomProps extends CommentVotingComponentProps<CommentsList | messageListFragment> {
   voteProps: VotingProps<VoteableTypeClient>;
+  invertColors?: boolean;
```

Comment:
(idk what this is doing or why it's shared with `NamesAttachedReactionsCommentBottomProps`, for comment-related stuff?)